[2022-10-24T21:32:26.860+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: sales_datawarehouse.get_vendas manual__2022-10-24T21:32:22.616847+00:00 [queued]>
[2022-10-24T21:32:26.876+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: sales_datawarehouse.get_vendas manual__2022-10-24T21:32:22.616847+00:00 [queued]>
[2022-10-24T21:32:26.877+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-10-24T21:32:26.878+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2022-10-24T21:32:26.878+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-10-24T21:32:26.903+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): get_vendas> on 2022-10-24 21:32:22.616847+00:00
[2022-10-24T21:32:26.909+0000] {standard_task_runner.py:54} INFO - Started process 21752 to run task
[2022-10-24T21:32:26.913+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'sales_datawarehouse', 'get_vendas', 'manual__2022-10-24T21:32:22.616847+00:00', '--job-id', '382', '--raw', '--subdir', 'DAGS_FOLDER/sales_dw.py', '--cfg-path', '/tmp/tmp6qndxse7']
[2022-10-24T21:32:26.915+0000] {standard_task_runner.py:83} INFO - Job 382: Subtask get_vendas
[2022-10-24T21:32:26.916+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/sales_dw.py
[2022-10-24T21:32:27.602+0000] {task_command.py:384} INFO - Running <TaskInstance: sales_datawarehouse.get_vendas manual__2022-10-24T21:32:22.616847+00:00 [running]> on host 83cd0ab98720
[2022-10-24T21:32:27.684+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=pedromenegat
AIRFLOW_CTX_DAG_ID=sales_datawarehouse
AIRFLOW_CTX_TASK_ID=get_vendas
AIRFLOW_CTX_EXECUTION_DATE=2022-10-24T21:32:22.616847+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-24T21:32:22.616847+00:00
[2022-10-24T21:32:27.697+0000] {base.py:71} INFO - Using connection ID 'banco_vendas' for task execution.
[2022-10-24T21:32:30.038+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-10-24T21:32:30.057+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=sales_datawarehouse, task_id=get_vendas, execution_date=20221024T213222, start_date=20221024T213226, end_date=20221024T213230
[2022-10-24T21:32:30.098+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-10-24T21:32:30.144+0000] {local_task_job.py:273} INFO - 2 downstream tasks scheduled from follow-on schedule check
